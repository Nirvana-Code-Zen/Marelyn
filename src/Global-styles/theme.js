export default {
  colors: {
    primary: '#d4a413',
    primary_complement: '#efb810',
    primary_second_complement: '#f3c332',
    secondary: '#155db1',
    secondary_complement: '#2196f3',
    error: '#D50000',
    white: '#ffffff',
    hover_search: '#efb810'
  },
  fonts: {
    texts_titles: '"Didact Gothic", sans-serif'
  },
  fonts_sizes: {
    titles: '50px',
    subtitles: '40px',
    small_subtitles: '30px',
    mobile_font: '25px',
    texts: '19px',
    medium_text: '17px',
    button_text: '16px',
    small_texts: '15px',
    extra_small_texts: '14px'
  },
  width_sizing: {
    wTen: '0.625rem', /* 10px */
    wTwelve: '0.75rem', /* 12px */
    Fourteen: '0.875rem', /* 14px */
    wFourteen: '1rem', /* 16px */
    wTwenty: '1.25rem', /* 20px */
    wTwentyFour: '1.5rem', /* 24px */
    wTwentyEight: '1.75rem', /* 28//px */
    wThirtyTwo: '36rem', /* 32px */
    wSixty: '160px',
    wThirteenthTundred: '300px'
  },
  width_porcentages: {
    wOneSixth: '16%',
    wOneFifth: '20%',
    wQuarter: '25%',
    wOneThrid: '33.333333%',
    wTwoFifths: '40%',
    wMedium: '50%',
    wTreeFifths: '60%',
    wTwoThrid: '66.666667%',
    wFourFifths: '80%',
    wTreeQuarters: '75%',
    wFiveSixths: '83.333333%',
    wElevenTwelfth: '91.666667%',
    wComplet: '100%'
  },
  width_creen: {
    wScreen: '100vw',
    wMin: 'min-content',
    wMax: 'max-content',
    wFit: 'fit-content',
    minW0: '0px'
  }
}
