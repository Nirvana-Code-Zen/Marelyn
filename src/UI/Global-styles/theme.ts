interface Theme {
  colors: {
    primary: string;
    primary_complement: string;
    primary_second_complement: string;
    secondary: string;
    secondary_complement: string;
    error: string;
    white: string;
    hover_search: string;
    texts: string;
    texts_secondary: string;
    hover_tabs: string;
  };
  fonts: {
    texts_titles: string;
  };
  fonts_sizes: {
    titles: string;
    subtitles: string;
    medio_titles: string;
    small_subtitles: string;
    texts: string;
    medium_text: string;
    button_text: string;
    small_texts: string;
    extra_small_texts: string;
  };
  width_sizing: {
    [key: string]: string; 
  };
  width_porcentages: {
    [key: string]: string;
};
width_creen:{
  [key:string]: string;
}
}


export const theme: Theme = {
  colors: {
    primary: '#d4a413',
    primary_complement: '#efb810',
    primary_second_complement: '#f3c332',
    secondary: '#155db1',
    secondary_complement: '#2196f3',
    error: '#D50000',
    white: '#ffffff',
    hover_search: '#efb810',
    texts: '#999999',
    texts_secondary: '#0B0A59',
    hover_tabs: '#E3D99F'
  },
  fonts: {
    texts_titles: '"Didact Gothic", sans-serif'
  },
  fonts_sizes: {
    titles: '30.12rem', /* 50px */
    subtitles: '2.5rem', /* 40px */
    medio_titles: '1.87rem', /* 30px */
    small_subtitles: '1.25rem', /* 20px */
    texts: '1.18rem', /* 19px */
    medium_text: '1.06rem', /* 17px */
    button_text: '1rem', /* `16px */
    small_texts: '0.93rem', /* 15px */
    extra_small_texts: '0.87rem'/* 14px */
  },
  width_sizing: {
    wTen: '0.625rem', /* 10px */
    wTwelve: '0.75rem', /* 12px */
    Fourteen: '0.875rem', /* 14px */
    wFourteen: '1rem', /* 16px */
    wTwenty: '1.25rem', /* 20px */
    wTwentyFour: '1.5rem', /* 24px */
    wTwentyEight: '1.75rem', /* 28//px */
    wThirtyTwo: '36rem', /* 32px */
    wSixty: '160px',
    wThirteenthTundred: '300px'
  },
  width_porcentages: {
    wOneSixth: '16%',
    wOneFifth: '20%',
    wQuarter: '25%',
    wOneThrid: '33.333333%',
    wTwoFifths: '40%',
    wMedium: '50%',
    wTreeFifths: '60%',
    wTwoThrid: '66.666667%',
    wFourFifths: '80%',
    wTreeQuarters: '75%',
    wFiveSixths: '83.333333%',
    wElevenTwelfth: '91.666667%',
    wComplet: '100%'
  },
  width_creen: {
    wScreen: '100vw',
    wMin: 'min-content',
    wMax: 'max-content',
    wFit: 'fit-content',
    minW0: '0px'
  }
}

