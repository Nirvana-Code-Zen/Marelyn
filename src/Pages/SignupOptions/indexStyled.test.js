import { ThemeProvider } from 'styled-components'
import renderer from 'react-test-renderer'
import SignUpOptionsStyled from './indexStyled'

import theme from '../../Global-styles/theme'

describe('<SignUpOptionsStyled/>', () => {
  test('Should be global styles ready', () => {
    const tree = renderer.create(
      <ThemeProvider theme={theme}>
        <SignUpOptionsStyled />
      </ThemeProvider>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
