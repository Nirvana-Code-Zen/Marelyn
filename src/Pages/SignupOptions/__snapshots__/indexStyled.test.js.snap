// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUpOptionsStyled/> Should be global styles ready 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  width: 400px;
  height: 254px;
  margin: 10rem auto;
  padding: 1em;
  background: #F4F6FB;
  border: 1px solid white;
  box-shadow: 10px 10px 64px 0px rgba(180,180,207,0.75);
}

.c0 svg {
  color: #d4a413;
  margin-bottom: 20px;
  margin-right: 5px;
}

<div
  className="c0"
/>
`;
